### ISSUE TITLE
execheap + errorcode 133

### ISSUE THREAD

**Issue by dabzse** [VSCodium/vscodium#1962]
**Describe the bug**
as you can see on the screenshots, first appeared in 25 December 2023. I can't remember if I used Fedora 38 or 39, but this is not the important thing at the moment.
it gave few warnings only. but increased up to 30k+. few days ago it was about 35k.
and then the "slaughter" began... so I just started recording...

about the screen record:
it is cutted on more places, and because of the size > 10MB, I had to split into 2 pieces, but believe me, these short versions are really enough.
the `execheap` warning increased very fast. I know, there was less, than 40k, but after 30-35 minutes it became 60k+. that's too much.
for me it's "okay", my system caught it. but what about the others?
and the error code: 133 -> I guess it is a python extension bug, but there is no active python extension in that profile. I was able to record that too. it wasn't easy.

source process always shows: code, but comm shows: codium

comm="codium|code|chrome"
the common thing in these apps: the `electronjs` base. possible there is a bug...

(about chrome: sidekick uses the name chrome. don't ask me why)

**Please confirm that this problem is VSCodium-specific**
- [ ] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.
  - I have vscode as well, just because of the university, but unfortunately I have to keep it. I don't use it since the end May.  (the end of semester)
  - the extension_dir is not the same, because of the telemetry.

**Please confirm that the issue/resolution isn't already documented**
- [ ] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.
- I asked the project leader, he sent me a message to report it.

**To Reproduce**
sometimes works well. but if not, see the screen record. (I had to cut into 2 parts and to resize because of the size > 10MB)
[VSCodium__errorcode_133__20240715_1825x_part1.webm](https://github.com/user-attachments/assets/5a071e6b-a89d-4e59-884a-c6223aeb83c0)
[VSCodium__errorcode_133__20240715_1825x_part2.webm](https://github.com/user-attachments/assets/efa0e08a-2af7-4abb-b9c8-44a193a8f70f)

**Expected behavior**

**Screenshots**
![execheap__20240712](https://github.com/user-attachments/assets/219c0e0b-b447-45ac-83f7-ef363e8d0102)
![execheap__20240714](https://github.com/user-attachments/assets/5915c94a-295b-4c9e-81e0-14b9d20d8162)
![execheap](https://github.com/user-attachments/assets/750ca66d-c03e-4dec-a8ef-9078b70a9fd0)
![execheap](https://github.com/user-attachments/assets/498dc1cd-c024-4482-b3f3-4d726cef66c1)
![warnings](https://github.com/user-attachments/assets/f2d95fdf-e294-4cbe-afce-35395675655e)
![execheap__20240718](https://github.com/user-attachments/assets/36d41e14-f26c-4c2f-aec1-41d6c723527d)

**Desktop (please complete the following information):**
 - OS: [Fedora 40]
 - Architecture [x64]
 - Version [1.91.1] - latest, as always when it possible
 - App Manager [RPM]
 - Sandboxed [no]

**Additional context**
just opened VSCodium to see the version number. alarm increased to 95844
before sending the bug report, there were more...

**Comment by dabzse** [comment]
news:
brave-browser-nightly-1.70.3-1.x86_64
updated few minutes ago, I just launched it, and got the execheap alarm/report...
surely not just VSCodium related bug. but this is more than that :(
![execheap_20240718_brave](https://github.com/user-attachments/assets/ffe201c1-be0c-42f2-9d2a-d53923ea42e0)

**Comment by daiyam** [comment]
It's a flatpak issue (https://github.com/flathub/com.vscodium.codium/issues/345)

**Comment by dabzse** [comment]
but I have installed from RPM package, and every time when is possible, I'm using the RPM package (for every apps);
I don't like flatpak, nor snap. AppImage and .tar.gz or .taz.bz2 or tar.xz (to unpack and run) are so-so, but if there is RPM, then that's the winner.
but if the problem with flatpak, then why some other apps give the same warning? there was no brave until today....

**Comment by dabzse** [comment]
and I'm pretty sure: if I will install chrome and will add uBlockOrigin, it will crash the same way.

but anyways, this is not only a VSCodium related bug (this is much bigger). you can close it if you think....
thank you, and I'm sorry about this.
happy coding :-)

**Comment by daiyam** [comment]
Yep, rpm seems to be also affected (https://github.com/VSCodium/vscodium/issues/1931)

**Comment by dabzse** [comment]
thanks. and what about the error code 133? (check the screen record)

**Comment by crimsonfall** [comment]
I've experienced these Execheap issues as well over on the Flatpak version running on Fedora. These SELinux alerts also affect all Electron and Chromium-based apps as well, no matter how they are packaged (hence your issues with Chrome).

Apparently, it was a kernel bug all along, as indicated in [this Bugzilla discussion](https://bugzilla.redhat.com/show_bug.cgi?id=2254434). The fix was merged to the mainline kernel, so at this point, all we have to do is wait for Fedora to update the kernel to their repositories.

**Comment by dabzse** [comment]
thanks @crimsonfall !
so I had right there is a problem with `electronjs` based apps. I didn't thought for a kernel bug.
but anyways: thanks for a link too!
